test/clj/cider/nrepl/middleware/resource_test.clj

```diff
@@ -19,17 +18,3 @@
                        (:resources-list response))))
       (is (seq (filter #(re-matches #".*test/resources/test\.txt" (:file %))
                        (:resources-list response)))))))
-
-(deftest resource-op-error-handling-test
```

Comment:
Probably we should keep the error handling tests.

Reply:
Uhm, I have checked that and it is `redef`-ing an api that is in `orchard` so I am not sure it still makes sense to test it?

Reply:
The point of those tests was not to test something in the API, but that unexpected errors are properly handled. Basically it tests that the middleware is using `with-safe-transport`. As if you've inlined the reply functions that's a bit hard to do today, though. Anyways, not a big deal.

Reply:
Oh I see, good to know, it would be good to test these cross cutting things in some more generic way but yeah .. not easy with middlewares maybe? Dunno in any case thanks for merging!